pkg/controllers/workapplier/controller_test.go

```diff
@@ -94,6 +94,20 @@ var (
 	nsUnstructured *unstructured.Unstructured
 	nsJSON         []byte
 
+	configMap = &corev1.ConfigMap{
```

Comment:
where is this used?

Reply:
Hi Ryan! This is used for the newly added decoding error test; for negative cases configMaps are a bit easier to handle than deployments.